### PR TITLE
remove audit.k8s.io/v1[alpha|beta]1 versions

### LINKED ISSUE #98035: Deprecate and Remove audit.k8s.io/v1[alpha|beta]1 Versions
**What would you like to be added**:
Since k8s v1.12 the API audit.k8s.io has a stable version V1, ref: https://github.com/kubernetes/kubernetes/pull/65891.
However, the v1alpha1 and v1beta1 versions of the API are still not deprecated and available
- https://github.com/kubernetes/kubernetes/tree/v1.21.0-alpha.1/staging/src/k8s.io/apiserver/pkg/apis/audit/v1alpha1
- https://github.com/kubernetes/kubernetes/tree/v1.21.0-alpha.1/staging/src/k8s.io/apiserver/pkg/apis/audit/v1beta1

If they got removed, the kube-apiserver flags `--audit-log-version` and `--audit-webhook-version` will accept only single valid value `audit.k8s.io/v1`, so these flags can be also deprecated at the moment, however I lean toward keeping them in case a new API version is implemented in the future.

/area audit
/sig auth
/kind deprecation
/kind cleanup
/kind api-change

### PR DESCRIPTION
**Pull request by carlory**
#### What type of PR is this?

/kind cleanup
/kind feature
/kind api-change

#### Which issue(s) this PR fixes:

Fixes #98035

#### Does this PR introduce a user-facing change?

```release-note
kube-apiserver: --audit-log-version and --audit-webhook-version now only support the default value of audit.k8s.io/v1. The v1alpha1 and v1beta1 audit log versions, deprecated since 1.13, have been removed.
```

### CODE REVIEW

**Comment by fedebongio** [comment]
Hi @carlory we triaged this one, and wondered if it would be good to discuss in the next SIG meeting. I'm adding it to the agenda.

**Comment by carlory** [comment]
@fedebongio yeah，I will do it this week.

**Comment by carlory** [comment]
@fedebongio It's ready for review.

File: staging/src/k8s.io/apiserver/pkg/audit/policy/reader_test.go [hunk 1]
```diff
@@ -90,7 +90,7 @@ var expectedPolicy = &audit.Policy{
 }
 
 func TestParser(t *testing.T) {
-	for _, version := range []string{"v1", "v1alpha1", "v1beta1"} {
+	for _, version := range []string{"v1"} {
```
Comment (ritazh) [inline staging/src/k8s.io/apiserver/pkg/audit/policy/reader_test.go:93]: nit: do we still need the array if it's just one item?

**Comment by fedebongio** [comment]
We discussed in the SIG meeting today, the full recording is here: https://youtu.be/1f-LRZV6U9Y

/assign @deads2k

File: staging/src/k8s.io/apiserver/pkg/server/options/audit.go [hunk 2]
```diff
@@ -235,8 +233,6 @@ func validateBackendBatchOptions(pluginName string, options AuditBatchOptions) e
 }
 
 var knownGroupVersions = []schema.GroupVersion{
-	auditv1alpha1.SchemeGroupVersion,
```
Comment (deads2k) [inline staging/src/k8s.io/apiserver/pkg/server/options/audit.go:238]: note to self for future me.  This is used a few lines down in a validate method and will prevent the server from starting if the value is anything except v1.

**Comment by pacoxu** [comment]
/lgtm cancel
To verify test failures, you need to run the `make update`.

**Pull request merged**